Description: Removed some unnecessary test setup such as SimpleCov and
  minitest-reporters.
diff --git a/test/helper.rb b/test/helper.rb
index ed3133c6..4f5d4f1b 100644
--- a/test/helper.rb
+++ b/test/helper.rb
@@ -13,22 +13,10 @@
 # - NOKOGIRI_GC: read more in test/test_memory_leak.rb
 #
 
-require "simplecov"
-SimpleCov.start do
-  add_filter "/test/"
-  enable_coverage :branch
-end
-
 $VERBOSE = true
 
 require "minitest/autorun"
 require "minitest/benchmark"
-require "minitest/reporters"
-
-nokogiri_minitest_reporters_options = { color: true, slow_count: 10, detailed_skip: false }
-nokogiri_minitest_reporters_options[:fast_fail] = true if ENV["NOKOGIRI_TEST_FAIL_FAST"]
-puts "Minitest::Reporters options: #{nokogiri_minitest_reporters_options}"
-Minitest::Reporters.use!(Minitest::Reporters::DefaultReporter.new(nokogiri_minitest_reporters_options))
 
 require "fileutils"
 require "tempfile"
